README.md

```diff
@@ -1,2 +1,6 @@
 # discord-rpg-experiment
+
+[![Javscript CI](https://github.com/TerrenceJChan/discord-rpg-game/actions/workflows/javascript.yml/badge.svg)](https://github.com/TerrenceJChan/discord-rpg-game/actions/workflows/javascript.yml)
+[![Database CI](https://github.com/TerrenceJChan/discord-rpg-game/actions/workflows/database.yml/badge.svg)](https://github.com/TerrenceJChan/discord-rpg-game/actions/workflows/database.yml)
```

Comment:
Fun last step: stick these in the README (you can generate them on the actions page) to show the status of the last run!